src/PDFA-3-0/Cert/XIS-Server/medmij-pdfa-xis-2-3-serve-1-documentmanifest.xml

```diff
@@ -19,4 +19,13 @@
         <nts:include value="assert.response.numResources" scope="common"
             resource="DocumentManifest" count="1"/>
     </test>
+    
+    <test id="scenario2-3-serve-1-DocumentManifest" nts:in-targets="NoManifest">
+        <name value="Scenario 2.3"/>
+        <description value="Handle unsupported request for DocumentManifest resources"/>
+        <nts:include value="medmij/test.xis.search" scope="common"
+            resource="DocumentManifest"
+            params="?status=current"/>
```

Comment:
Should `params` be equal to `?status=current&amp;created=ge${DATE, T, D,-365}&amp;created=le${DATE, T, D,-60}` (see also scenario 2.4, or scenario 2.3 for other targets)?

Reply:
Good catch, I fixed this. I also removed the two build files.